src/Network/OAuth.hs

```diff
@@ -55,18 +55,14 @@ module Network.OAuth (
 
   ) where
 
-import qualified Crypto.Random                   as R
 import qualified Network.HTTP.Client             as C
 import qualified Network.OAuth.Signing           as S
 import qualified Network.OAuth.Types.Credentials as O
 import qualified Network.OAuth.Types.Params      as O
 
--- | Sign a request with a fresh set of parameters. Creates a fresh
--- 'R.SystemRNG' using new entropy for each signing and thus is potentially
--- /dangerous/ if used too frequently. In almost all cases, 'S.oauth'
--- should be used instead.
+-- | Sign a request with a fresh set of parameters. Uses @MonadRandom IO@, getting
+-- new entropy for each signing and thus is potentially /dangerous/ if used too
+-- frequently. In almost all cases, 'S.oauth' should be used instead with a suitably
+-- seeded PRNG.
 oauthSimple :: O.Cred ty -> O.Server -> C.Request -> IO C.Request
-oauthSimple cr srv req = do
-  entropy   <- R.createEntropyPool
-  (req', _) <- S.oauth cr srv req (R.cprgCreate entropy :: R.SystemRNG)
-  return req'
+oauthSimple = S.oauth
```

Comment:
PR #17 simply removes this now-trival wrapper. I kept it mostly because the comment help to clarify what could otherwise be a confusing and tricky issue.